wiki/Gameplay/Game_modifier/Flashlight/en.md

```diff
@@ -61,7 +61,9 @@ When paired with the Hidden mod, the fruits are momentarily visible if the catch
 
 ### osu!mania
```

Comment:
quick question

would it be fine to add the distinguish hatnote (<pretext><code>*Not to be confused with Fade In (Mod)*</code></pretext>) after the header or should it just be kept at the top?

Reply:
bump

Reply:
I have no preference -- maybe rephrase it slightly if you keep it at the top?

```md
*Not to be confused with [Hidden (mod)](/wiki/Gameplay/Game_modifier/Hidden) or osu!mania's [Fade In (Mod)](/wiki/Gameplay/Game_modifier/Fade_In).*
```